decidim-elections/app/models/decidim/elections/question.rb

```diff
@@ -12,6 +12,13 @@ class Question < ApplicationRecord
       belongs_to :election, foreign_key: "decidim_elections_election_id", class_name: "Decidim::Elections::Election", inverse_of: :questions
       has_many :answers, foreign_key: "decidim_elections_question_id", class_name: "Decidim::Elections::Answer", inverse_of: :question, dependent: :destroy
 
+      has_many :ballot_style_questions,
+               class_name: "Decidim::Votings::BallotStyleQuestion",
+               foreign_key: :decidim_elections_question_id,
+               inverse_of: :question,
+               dependent: :delete_all
+      has_many :ballot_styles, through: :ballot_style_questions
```

Comment:
💬 Do you really need to add this two associations? I'd prefer to keep the coupling between elections and votings as minimal as possible, especially the knowledge of votings from elections.

Reply:
I initially avoided doing this but the it felt a bit...hypocrite? I mean, the DB relationship is there, we'd just pretend it does not exist in the model.
But I get your point and will follow your suggestion!